fpt/src/ppu.rs

```diff
@@ -78,6 +83,10 @@ impl Ppu {
     fn oam_scan(&mut self) {
         if self.dots_this_frame % 456 == (80 - 1) {
             self.tilemap = self.bus.with_vram(VRamContents::load);
+            let oam = self.bus.copy_range(map::OAM);
+            self.sprites = (0..40)
+                .map(|sprite_index| Sprite::load(&oam[sprite_index * 4..(sprite_index * 4 + 4)]))
```

Comment:
can you explain the magic numbers here? the * 4 and + 4.
the ppu already has lots of magic numbers I introduced that I dislike 😅